tests/test_strict.py

```diff
@@ -8,22 +11,39 @@
 
 
 @pytest.mark.parametrize(
-    'strict,input_value,expected',
+    'strict_to_validator,strict_in_schema,input_value,expected',
     [
-        (False, 123, 123),
-        (False, '123', 123),
-        (None, 123, 123),
-        (None, '123', 123),
-        (True, 123, 123),
-        (True, '123', Err('Input should be a valid integer [type=int_type')),
+        (False, False, 123, 123),
+        (False, False, '123', 123),
+        (None, False, 123, 123),
+        (None, False, '123', 123),
+        (True, False, 123, 123),
+        (True, False, '123', Err('Input should be a valid integer [type=int_type')),
```

Comment:
This was the only case I had to edit after exploding the parametrization. Passing `strict=True` at validation time overrides whatever is set on the schema. Good to know!